oeps/architectural-decisions/oep-0014-proc-archive-repos.rst

```diff
@@ -74,17 +74,17 @@ First, if the repository is public, and a part of Open edX releases, follow thes
 Archive Steps
 =============
 
-These steps should be followed for all repos within the Open edX organization (forks included). After some experiments with keeping archived repos in the ``openedx`` organization, we've learned that having abandoned code show up in searches hinders work to understand the current state of the system and the risk around new work, particularly deprecations and API changes. Thus we decided to move all archived repositories to a separate org.
+These steps should be followed for all repos within the Open edX organization (forks included). After some experiments with keeping archived repos in the `openedx github organization`_, we've learned that having abandoned code show up in searches hinders work to understand the current state of the system and the risk around new work, particularly deprecations and API changes. Thus we decided to move all archived repositories to a separate org.
 
 1. Update the README.rst file in the repository to add a brief note about why the repo is being archived, and what is serving as its replacement (where applicable). This may be as simple as a linking to the appropriate DEPR ticket.
 
 2. Unless you have the relevant permissions to perform this step, create a new `GitHub Request on the tCRIL board`_  and ask them to do the following:
 
-   - Archive the repository per `GitHub's archive process`_
+  - Archive the repository per `GitHub's archive process`_
 
-   - Move the repository to the openedx-unsupported organization
+  - Move the repository to the openedx-unsupported organization
 
-      - If the repo is not coming from the `openedx github org`_ then before moving it, rename it with a prefix of the source org's name. For example the ``notifier`` repo in the ``edx-solutions`` org wolud be renamed to ``edx-solutions-notifier`` before moving.
```

Comment:
**Note to Reviewers:** I don't think tCRIL has any orgs other than `openedx`, so I removed this note and moved it to the 2U/edX private runbook. If we want something here for tCRIL or other community members, we can add a note, but it seemed to muddy the instructions if it isn't needed.